src/Imports/Import.php

```diff
@@ -83,11 +84,16 @@ public function getReplacements(Collection $rows): array
     {
         $replacements = [];
 
+        /** @var Field[] $relations */
         $relations = collect($this->fields)->filter->isRelation();
 
         // author, category, etc
         foreach ($relations as $relation) {
-            $relationshipName = Str::camel($relation->key);
+            if (!empty($relation->internal)) {
+                $relationshipName = $relation->internal;
+            } else {
+                $relationshipName = Str::camel($relation->key);
```

Comment:
Let's remove this camel incase they want the relation in a different format.

And add the if internal is null check to `init` and setting $internal if null

Reply:
```php
Field::make('category_name', 'category.name')
```

Reply:
This might effect #65. Sorry it wasnt more clear

Reply:
Sorry @dillingham not able to understand the comment here :(

Reply:
@mitulgolakiya no problem :) I was half awake when I explained it

Here was what I was going for: https://github.com/headlesslaravel/formations/pull/67/files